lib/rubocop/cop/rspec/described_class_module_wrapping.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Avoid opening modules and defining specs within them.
+      #
+      # @example
+      #   # bad
+      #   module MyModule
+      #     RSpec.describe MyClass do
+      #       # ...
+      #     end
+      #   end
+      #
+      #   # good
+      #   RSpec.describe MyModule::MyClass do
+      #     # ...
+      #   end
+      #
+      # @see https://github.com/rubocop-hq/rubocop-rspec/issues/735
+      class DescribedClassModuleWrapping < Cop
+        MSG = 'Avoid opening modules and defining specs within them.'
+
+        def_node_matcher :rspec_block?,
```

Comment:
Can be `def_node_search`, and in this case `node.each_descendant` is unnecessary.

Reply:
Noted. I'll give that a shot!